src/components/datepicker/DatePickerInput.js

```diff
@@ -1,20 +1,19 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import Box, { omitBoxProps, pickBoxProps } from '../box';
-import DayPickerInput from 'react-day-picker/DayPickerInput';
+import Box, { pickBoxProps } from '../box';
+import DatePicker from '../datepicker';
 import Icon from '../icon';
-import NavigationBar from './NavigationBar';
-import WeekDay from './WeekDay';
-import { convertModifiersToClassnames } from './utils';
-import { IconCalendarSmallOutline } from '@teamleader/ui-icons';
-import ValidationText from '../validationText';
+import Input from '../input';
+import Popover from '../popover';
 import cx from 'classnames';
-import omit from 'lodash.omit';
 import theme from './theme.css';
+import LocalUtils, { formatDate } from './localUtils';
```

Comment:
Why `localUtils` instead of `localeUtils`?

Reply:
Damn, typo 🙈